NitroxClient/MonoBehaviours/AnimationController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace NitroxClient.MonoBehaviours
+{
+    public class AnimationController : MonoBehaviour
+    {
+        public static readonly float SMOOTHING_SPEED = 0.95f;
```

Comment:
What will need to be adjusted for latency is the movement helper, but this variable here just moves the legs back down when the player slows down swimming.